.changeset/olive-boxes-hide.md

```diff
@@ -0,0 +1,13 @@
+---
+'@backstage/backend-plugin-api': minor
+'@backstage/backend-test-utils': minor
+'@backstage/plugin-scaffolder-backend': minor
+'@backstage/backend-defaults': minor
+'@backstage/plugin-catalog-backend': minor
+'@backstage/plugin-scaffolder-node': minor
```

Comment:
I realize it's useful to see usage patterns, but the review gets a bit riskier when having to check the individual plugin usages too, in addition to the core service design.